CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 10.0
+
+* Moved out tester classes to the dedicated test-fixtures artifact
```

Comment:
@damien-urruty-sonarsource I updated the CHANGELOG after your approval, are you fine with this change? (and does it fit the 'interfaces changes only' approach? I think so, as this is potentially breaking change, in the end, it's a separate artefact)

Reply:
yep, it's all good!